modules/cnf-working-with-fast-event-notifications-api.adoc

```diff
@@ -0,0 +1,283 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-working-with-fast-event-notifications-api_{context}"]
+= Working with the PTP fast event notifications REST API
+
+You can use the PTP fast event notifications REST API to subscribe an application to the PTP events that are generated for the parent node. PTP fast events notifications are available on each node where a PTP capable network interface is configured.
+
+You can subscribe DU applications to PTP event notifications using the resource address `/cluster/node/<node_name>/ptp`, where `<node_name>` is the cluster node running the DU application.
```

Comment:
```suggestion
You can subscribe DU applications to PTP event notifications by using the resource address `/cluster/node/<node_name>/ptp`, where `<node_name>` is the cluster node running the DU application.
```

> ! using _noun_
To avoid ambiguity, replace this gerund with either "by using" or "that uses".

I see this a few other places, so please check.

Reply:
Thanks, fixed these.